### PR TITLE
Add esbuild

### PR DESCRIPTION
**Pull request by geoffharcourt**
https://esbuild.github.io/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/esbuild.png" width="20" height="20" alt="esbuilt"/> | `:esbuild:`
```
Comment (jfhinchcliffe) [inline README.md:30]: tiny typo - s/esbuilt/esbuild .

**Comment by jfhinchcliffe** [comment]
Thanks for the PR, @geoffharcourt . Once that typo is fixed we can approve and merge 😁

**Review by jfhinchcliffe — APPROVED** [review]
Yeeha! Thanks @geoffharcourt You'll see this on Buildkite.com in a few days.

**Comment by geoffharcourt** [comment]
Woot! Thanks @jfhinchcliffe!

**Pull request merged**